src/Autocomplete/assets/src/controller.ts

```diff
@@ -295,7 +304,7 @@ export default class extends Controller {
             },
             optgroupField: 'group_by',
             // avoid extra filtering after results are returned
-            score: (search: string) => (item: any) => 1,
+            score: (_search: string) => (_item: any) => 1,
```

Comment:
why this change ? genuine question here

Reply:
The underscore is used to indicate an unused variable while making the linter happy.

Reply:
Was he unhappy ? never observed that .. but ok :)

Reply:
🤷🏻‍♂️ It's not easy being a linter I guess.